openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -94,10 +92,16 @@ window.Promise = Promise;
 
 // Initialise some things
 jQuery(function () {
+    const $markdownTextAreas = $('textarea.markdown');
     initValidate($);
     autocompleteInit($);
     addNewFieldInit($);
     automaticInit($);
+    // wmd editor
+    if ($markdownTextAreas.length) {
```

Comment:
Is there a reason this is here? Can we place it next to the other dynamic js import? It's current position makes me think one of the following functions depends on this happening here.

Reply:
Just defensive programming - since the code was in automatic before, I wanted to leave its execution in around about the same place. Eventually this folder will all be imports.